.travis.yml

```diff
@@ -9,16 +9,7 @@ addons:
 
 matrix:
     include:
-        - php: hhvm
-        - php: 5.3.3
-        - php: 5.3
-        - php: 5.4
-        - php: 5.5
         - php: 5.6
-          env: deps=low
-        - php: 5.6
-          env: deps=high
-        - php: nightly
```

Comment:
you should re-enable them to be sure your wrapper script works on all PHP versions, as you are also changing the Travis setup